src/main/java/com/bravo/user/dao/repository/PaymentRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.bravo.user.dao.repository;
+
+import com.bravo.user.dao.model.Payment;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface PaymentRepository extends JpaRepository<Payment, String> {
+
+    List<Payment> findAllByUserId(String userId);
```

Comment:
How is this different than `findByUserId`? Does `findByIgnoreCase` make sense here?

Reply:
I just use that more descriptive even though it is the same result as byUserId. And about the ignoreCase, I think we should look for exact keys, and also we could face some performance issues if we start doing searches with ignore case, let's say we have an index in the userId, as far as I am concerned this type of searches would ignore the index and probably be an slow query.